src/helper/stringHelper.js

```diff
@@ -0,0 +1,6 @@
+const isNotEmptyString = (data, trim = false) =>
+	typeof data === 'string' && ((trim && data.trim().length > 0) || data.length > 0);
```

Comment:
this fails, if trim is true, then the last condition will work anyway

Reply:
thank you :) that is right i miss for the last part the `!trim` that wherer requeierd.